hack/.binEnv

```diff
@@ -3,10 +3,26 @@
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 BIN_DIR="${SCRIPT_DIR}/../bin/"
 
+containerRuntime() {
```

Comment:
TODO: `.binEnv` vs `.quickstartEnv` - which comes first (remove the dupe `containerRuntime`)